lbry/dht/node.py

```diff
@@ -258,3 +258,13 @@ def accumulate_peers(self, search_queue: asyncio.Queue,
                          ) -> typing.Tuple[asyncio.Queue, asyncio.Task]:
         queue = peer_queue or asyncio.Queue(loop=self.loop)
         return queue, self.loop.create_task(self._accumulate_peers_for_value(search_queue, queue))
+
+
+async def prime_node_with_peer_addresses(peer_list: typing.List[typing.Tuple[str, int]], peer_queue: asyncio.Queue):
+    try:
+        peer_address_list = [(await resolve_host(url, port, proto='tcp'), port) for url, port in peer_list]
+        kademlia_peer_list = [make_kademlia_peer(None, address, None, tcp_port=port, allow_localhost=True)
+                              for address, port in peer_address_list]
+        peer_queue.put_nowait(kademlia_peer_list)
+    except asyncio.CancelledError:
+        pass
```

Comment:
why is the cancelled error trapped? usually trapping this error causes unexpected behavior.

Reply:
answer in the comments of this PR